File: Adivinha/app/src/main/java/annia/com/br/adivinha/MainActivity.java
package annia.com.br.adivinha;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    public static final String TITULO_APPBAR = "Jogo de Adivinhar";
    private Button botaoJogar;
    private TextView textoResultado;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        botaoJogar = findViewById(R.id.activity_button_jogar);
        textoResultado = findViewById(R.id.ativity_number_random);

        botaoJogar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Random randomico = new Random();

                int numeroAleatorio = randomico.nextInt(100);

                textoResultado.setText("Random Number " + numeroAleatorio);
            }
        });

        setTitle(TITULO_APPBAR);

    }
}

File: AdivinhaKotlin/app/src/main/java/com/example/adivinhakotlin/MainActivity.kt
package com.example.adivinhakotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private val APP_BAR:String = "Adivinha Kotlin"


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textoResultado:TextView = activity_textResultado
        val botaoJogar:Button = activity_button_play

        botaoJogar.setOnClickListener {
            val numeroAleatorio:Int = Random.nextInt(100)

            textoResultado.text = "Random Number: $numeroAleatorio"
        }

        setTitle(APP_BAR)

    }

}
